releases/release-6.3.0.md

```diff
@@ -0,0 +1,303 @@
+---
+title: TiDB 6.3.0 Release Notes
+---
+
+# TiDB v6.3.0 Release Notes
+
+Release date: xx xx, 2022
+
+TiDB version: 6.3.0-DMR
+
+In v6.3.0-DMR, the key new features and improvements are as follows:
+
+- TiKV and TiFlash support SM4 encryption at rest.
+- TiDB supports authentication with the SM3 algorithm.
+- The `CREATE USER` and `ALTER USER` statements support the `ACCOUNT LOCK/UNLOCK` option.
+- JSON data type and functions become generally available (GA).
+- TiDB supports Null Aware Anti Join.
+- TiDB provides execution time metrics at a finer granularity.
+- A new syntactic sugar is added to simplify Range partition definitions.
+- Range Columns partitioning supports defining multiple columns.
+
+## New features
+
+### SQL
+
+* Add a new syntactic sugar (Range INTERVAL partitioning) to simplify Range partition definitions [#35683](https://github.com/pingcap/tidb/issues/35683) @[mjonss](https://github.com/mjonss)
+
+    [Provides INTERVAL partitioning as a new way of defining Range partitions](/partitioned-table.md#range-interval-partitioning). You do not need to enumerate all partitions, which drastically reduces the lengthy way of writing Range partition statements. The semantic is equivalent to the original Range partition.
+
+* Range Columns partitioning supports defining multiple columns [#36636](https://github.com/pingcap/tidb/issues/36636) @[mjonss](https://github.com/mjonss)
+
+    Support [PARTITION BY RANGE COLUMNS (column_list)](/partitioned-table.md#range-columns-partitioning). `column_list` is no longer limited to a single column. The basic feature is the same as MySQL.
+
+* EXCHANGE PARTITION becomes GA [#35996](https://github.com/pingcap/tidb/issues/35996) @[ymkzpx](https://github.com/ymkzpx)
+
+    [EXCHANGE PARTITION](/partitioned-table.md#partition-management) becomes GA after performance and stability improvements.
+
+* TiDB supports two more [window functions](/tiflash/tiflash-supported-pushdown-calculations.md) [#5579](https://github.com/pingcap/tiflash/issues/5579) @[SeaRise](https://github.com/SeaRise) **tw：shichun-0415**
+
+    * `LEAD`
```

Comment:
```suggestion
    * `LEAD()`
```

Reply:
For this change, the feature owner and PM thought it would be more accurate if `LEAD` and `LAG` do not include `()` because they are used in the following three ways: 
> - LEAD/LAG(expr)
> - LEAD/LAG(expr, offset)
> - LEAD/LAG(expr1, offset, expr2)

Reply:
For me `LEAD()` would make it obvious that this is a function, without saying anything about the number and types of arguments.

Example: [the snprintf(3) man page](https://linux.die.net/man/3/snprintf) has this: "The functions in the printf() family produce output according to..." and "The functions snprintf() and vsnprintf() write at most size bytes...".

Reply:
LGTM